subprojects/scala/src/integTest/groovy/org/gradle/scala/compile/BasicScalaCompilerIntegrationTest.groovy

```diff
@@ -1,275 +0,0 @@
-/*
```

Comment:
There was a base class containing some tests for Scala compilation and one subclass for testing Scala compilation with Zinc. There were also some asserts ensuring that we use the right compiler (Ant/Zinc). Since there is only one way to compile Scala code in the Gradle code base it is not necessary anymore to keep the base class around and I inlined it. Checking which compiler is used does not make much sense, either.